app/jobs/release_object_job.rb

```diff
@@ -67,6 +63,34 @@ def create_release_tag(bulk_action, cocina, log)
     false
   end
 
+  # Return a copy of the existing cocina model with a new tag appended
+  def model_with_new_release_tag(cocina)
```

Comment:
Should this be using a change set?

Reply:
I don't think it needs to, but we could make a new change set object just for this sort of change. I think the indirection between having another class might make it slightly harder to follow than this simpler approach.

Reply:
I don't want to delay this ticket, so happy to have this discussion sync. Couldn't it just be added to an existing change set class?

Reply:
I'm not keen to have one set of functionality in a change set class that is used by one consumer and then a disjoint set of functionality in the same class used by another consumer.